src/libraries/System.Private.CoreLib/src/System/MemoryExtensions.cs

```diff
@@ -619,6 +680,9 @@ public static ReadOnlyMemory<char> AsMemory(this string? text, Range range)
                 case 3:
                     return IndexOfAnyExcept(span, values[0], values[1], values[2]);
 
+                case 4: // common for searching whitespaces
```

Comment:
Nit: (don't restart CI just for this)
```suggestion
                case 4: // common for searching ASCII whitespaces
```